### PR TITLE
[DOC] Add section on commit tagging to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by courajs**
2 questions:
- Does ember data use feature flags in the same way as ember.js?
- I left "submit your PR against `emberjs:master`", since people's remote in their `data` repo may be named `emberjs`. I could also change it to `data:master`

### CODE REVIEW

**Comment by courajs** [comment]
For #4238

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -99,10 +110,48 @@ taken straight from the Ruby on Rails guide:
...
+Documentation commits are tagged as `[DOC channel]` where channel is `canary`,
+`beta`, or `release`. If no release is provided `canary` is assumed. The channel should be the most stable release that this documentation change applies to.
+
+### Security
+
+Security commits will be tagged as `[SECURITY cve]`. Please do not submit security related PRs without coordinating with the security team. See the [Security Policy](http://emberjs.com/security/) for more information.
+
+### Other
+
+In general almost all commits should fall into one of these categories. In the cases where they don't please submit your PR untagged. An Ember contributor will let you know if tagging is required.
```
Comment (pangratz) [inline CONTRIBUTING.md]: Maybe we should use `An ember-data contributor will let ...` here

**Comment by pangratz** [comment]
> Does ember data use feature flags in the same way as ember.js?

Basically yes, the only difference is that `isEnabled` is imported from `ember-data/-private/features`;


> I left "submit your PR against emberjs:master", since people's remote in their data repo may be named emberjs. I could also change it to data:master

What do you thik of changing this to "submit your PR against the `master` branch of https://github.com/emberjs/data"? I think this should be clear enough where the create PR to...

---

Thank you so much @courajs for helping out here. Much appreciated!!!

@bmac @fivetanley since you are the masters of release ceremony, can you take a look here?

**Comment by courajs** [comment]
I opted to just say "against the `master` branch.", unless you think the url is useful.
They are reading this on the proper repo, after all

**Comment by bmac** [comment]
Looks great. Thanks @courajs

**Pull request merged**